conf/defaults.ini

```diff
@@ -593,8 +593,10 @@ enabled = true
 #################################### Internal Grafana Metrics ############
 # Metrics available at HTTP API Url /metrics
 [metrics]
-enabled           = true
-interval_seconds  = 10
+enabled              = true
+interval_seconds     = 10
+# Disable total stats (stat_totals_*) metrics to be generated
+disable_total_stats = false
```

Comment:
I think having enabled and disable directives in the same block might be confusing. Maybe `enable_total_stats` with a default of `true`?

Reply:
We want boolean settings to be default false. But can be discussed in this case.

Reply:
Ah that makes sense then.